src/main/java/org/sagebionetworks/bridge/hibernate/DateTimeZoneIdConverter.java

```diff
@@ -0,0 +1,21 @@
+package org.sagebionetworks.bridge.hibernate;
+
+import org.joda.time.DateTimeZone;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+@Converter
+public class DateTimeZoneIdConverter implements AttributeConverter<DateTimeZone, String> {
+    @Override
+    public String convertToDatabaseColumn(DateTimeZone dateTimeZone) {
+        if (dateTimeZone == null) return null;
+        return dateTimeZone.getID();
+    }
+
+    @Override
+    public DateTimeZone convertToEntityAttribute(String s) {
+        if (s == null) return null;
+        return DateTimeZone.forID(s);
+    }
+}
```

Comment:
This already exists as DateTimeZoneAttributeConverter

Reply:
I had an issue trying to get it to work with DateTimeZoneAttributeConverter. That seems to only work with offset strings because it uses bridge-base's DateUtils.parseZoneFromOffsetString(). So when it parses something like "America/Los_Angeles", it throws an error saying it needs to be in "+00:00" format.